llvm/lib/CodeGen/GlobalISel/CombinerHelper.cpp

```diff
@@ -7611,85 +7611,3 @@ bool CombinerHelper::matchFoldAMinusC1PlusC2(const MachineInstr &MI,
 
   return true;
 }
-
-bool CombinerHelper::matchUnmergeValuesAnyExtBuildVector(const MachineInstr &MI,
```

Comment:
It would be easier to review if the diff was local to the file, and this code wasn't moved to another file